src/vs/workbench/contrib/debug/browser/breakpointWidget.ts

```diff
@@ -141,6 +146,7 @@ export class BreakpointWidget extends ZoneWidget implements IPrivateBreakpointWi
 	fitHeightToContent(): void {
 		const lineNum = this.input.getModel().getLineCount();
 		this._relayout(lineNum + 1);
+		this.centerInputVertically();
```

Comment:
Does this end up calling `_doLayout` and does this end up calling then centerINputVertically twice?

Reply:
You are right. Removed this call to `centerInputVertically()`.